climada/engine/impact.py

```diff
@@ -1215,20 +1171,6 @@ def from_hdf5(cls, file_path: Union[str, Path]):
             ├─ event_name
             ├─ frequency
             ├─ imp_mat
```

Comment:
I think the `haz_type` is now missing from the description docstring. Should be in the `.attrs` ?

Reply:
True, looks like we are not writing it anymore. Will fix!

Reply:
It was written, but the test was not adapted properly. Everything works now!